solutions/gatekeeper-policies/naming-rules/project/setters.yaml

```diff
@@ -19,11 +19,22 @@ metadata:
   annotations:
     config.kubernetes.io/local-config: 'true'
 data:
-  #########
+  #########################
   # Project naming constraints values
-  #########
-  #
-  # Please change the values below
+  ##########################
   #
+  # Two-character code representing a client.
+  # Multiple clients can be supported by using a regex that evaluates multiple codes such as this: '^(aa|bb|cc)'
```

Comment:
Should we mention where to source these official codes from in order to standardize?

Reply:
could you provide more details, I think that when we decided to publish this constraint it was just to help other organization figure out a naming convention enforcement mechanism. If more information is given, we have to keep in mind the audience of that public package that is rather large.... my 2 cents..

Reply:
we can add the details to our internal instructions